mean_shift_sklearn.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+import argparse
+import matplotlib.pyplot as plt
+from itertools import cycle
+import numpy as np
+from sklearn.cluster import MeanShift
+
+information = """
+Calculates the mean_shift/peaks for the given dataset.
+
+Input:
+* bed6 file containing dataset
+* bandwidth (an integer value) to calculate the cluster radius/range
+* Output file name (optional)
+* Output_Graph file name (optional)
+
+Output:
+* bed6 file with new_centroids/peaks for the given data set.
+ default name "output.bed".
+* graph file, default name "graph.png".
+
+Example:
+- read file input.bed and write new peaks/centroids into output.bed
+- mean_shift.py input.bed 200 -o data_output.bed -g output-graph.png
+
+"""
+
+
+class Mean_Shift():
+
+    """
+    calcualtes the mean shift
+    >>> a = Mean_Shift(3)
+    >>> x = np.array([[1, 2, 3, 4, 5, 9, 10, 11, 12, 13, 20, 21, 22, 23, 24]])
+    >>> a.peak(x)
+    (array([[ 3.],
+           [11.],
+           [22.]]), 3)
+    """
+    def __init__(self, bandwidth):
+        self.bandwidth = bandwidth
+        self.labels = []
+
+    def peak(self, dt):
```

Comment:
dt is not a good variable name, it does not tell us what is in it

Reply:
add to function documentation: what kind of object is this? how to get it?

Reply:
done !!!